pkg/controller/ec2/routetable/controller.go

```diff
@@ -162,26 +161,16 @@ func (e *external) Create(ctx context.Context, mgd resource.Managed) (managed.Ex
 	if !ok {
 		return managed.ExternalCreation{}, errors.New(errUnexpectedObject)
 	}
-
-	cr.Status.SetConditions(runtimev1alpha1.Creating())
-	if err := e.kube.Status().Update(ctx, cr); err != nil {
-		return managed.ExternalCreation{}, errors.Wrap(err, errStatusUpdate)
-	}
```

Comment:
@muvaf just want to check and make sure I am understanding this correctly --> this status update is bumping resource version, which is causing our `meta.SetExternalName` update to fail, meaning we try to `Create` again on next reconcile and end up with two Route Tables?

Reply:
No, if status update works as expected, there is no problem because the local `cr` object would be updated to reflect the latest state and the next statements would work on the latest object, hence no error. However, api-server doesn't always return the updated object in that call for some reason that could be low limit, too busy or slow etcd.

Reply:
So we get a successful update but the returned object is not latest?

Reply:
For example, this problem happens in a constrained UbC environment 50% of the time.